# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../uclibc/patches/libtool-error_t.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---


Do not define error_t twice!
http://lists.gnu.org/archive/html/libtool-patches/2006-03/msg00017.html
  -- mtr

--- libtool-1.5.22/ltdl.m4.orig	2006-09-04 11:59:09.000000000 +0200
+++ libtool-1.5.22/ltdl.m4	2006-09-04 12:00:38.000000000 +0200
@@ -412,7 +412,9 @@
 AC_CHECK_TYPES([error_t],
   [],
   [AC_DEFINE([error_t], [int],
-    [Define to a type to use for `error_t' if it is not otherwise available.])],
+    [Define to a type to use for `error_t' if it is not otherwise available.])
+     AC_DEFINE([__error_t_defined], [1], [Define so that glibc/gnulib argp.h
+     does not typedef error_t.])],
   [#if HAVE_ARGZ_H
 #  include <argz.h>
 #endif])
